src/test/java/by/vadzimmatsiushonak/bank/api/facade/PaymentFacadeTest.java

```diff
@@ -184,4 +196,36 @@ public void initiatePaymentInsufficientSenderFunds() {
 
     }
 
+    @Nested
+    public class PaymentFacadeTestConfirmPayment {
+        @Test
+        public void confirmPayment() {
+            Boolean expectedConfirmed = true;
+            Confirmation confirmation = new Confirmation(CODE_INT, Map.of(ID, ID_LONG));
+
+            when(confirmationService.confirmCode(KEY, CODE_INT)).thenReturn(confirmation);
+            when(paymentService.findById(ID_LONG)).thenReturn(Optional.of(new BankPayment()));
```

Comment:
create variable for new BankPayment();

this will allow you to verify that paymentStatus was changed

example:
```
thenReturn(bankPayment)
assertEquals(ACCEPTED, bankPayment.getStatus)
```